doc/whats_new.rst

```diff
@@ -68,6 +68,9 @@ Changelog
   and :func:`~plot_spikes_raster` now plotted from 0 to tstop. Inputs tmin and tmax are deprecated,
   by `Katharina Duecker`_ in :gh:`769`
 
+- Use `np.isin()` in place of `np.in1d()` to address numpy deprecation,
+  by `Nick Tolley`_ in :gh:`799
```

Comment:
```suggestion
  by `Nick Tolley`_ in :gh:`799.
```

Reply:
should be `799`. adding the "`" after 799 as well, no? seems the use of the "." at the end is not done consistently in the doc

Reply:
shouldn't this go in the Bug section? It's not really a new feature